### PR TITLE
Allow withdrawing and viewing of the bitcoin balance on swap-cli

### PR DESCRIPTION
**Pull request by devbordecraft**
Same mechanics of the ASB but on swap-cli as discussed in #694

### CODE REVIEW

**Comment by rishflab** [comment]
hi thanks for the PR, this was pretty high priority on our list of things to do.

There are some formatting issues which can be fixed with
`dprint fmt`

install dprint
https://dprint.dev

**Comment by devbordecraft** [comment]
Fix is pushed

**Review by rishflab — APPROVED** [review]
Thanks again!

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - An issue where the ASB was unable to use the Monero wallet in case `monero-wallet-rpc` has been restarted.
   In case no wallet is loaded when we try to interact with the `monero-wallet-rpc` daemon, we now load the correct wallet on-demand.
   See issue https://github.com/comit-network/xmr-btc-swap/issues/652.
+- Add the ability to view the swap-cli bitcoin balance and withdraw
+  See issue https://github.com/comit-network/xmr-btc-swap/issues/694
```
Comment (thomaseizinger) [inline CHANGELOG.md:20]: This should go under `## Added` rather than `## Fixed`.

File: swap/src/cli/command.rs [hunk 2]
```diff
@@ -463,6 +529,23 @@ fn env_config_from(testnet: bool) -> env::Config {
...
+
+    if address.network != network {
+        bail!(BitcoinAddressNetworkMismatch {
+            expected: network,
+            actual: address.network
+        });
+    }
+
+    Ok(address)
+}
```
Comment (thomaseizinger) [inline swap/src/cli/command.rs:547]: We already have `validate_bitcoin_address`, lets use that instead.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Two minor things to be addressed. Can you also squash all commits into one while you are at it?

**Pull request merged**